quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java

```diff
@@ -45,15 +45,15 @@
     void shouldPingOpenAPI() {
         given()
             .header(ACCEPT, APPLICATION_JSON)
-            .when().get("/openapi")
+            .when().get("/q/openapi")
```

Comment:
That should be done also for `hero` and `fight` microservices

Reply:
This needs also be reported to the html pages...

Reply:
Done